src/plugins/intel_cpu/src/nodes/deconv.cpp

```diff
@@ -703,6 +703,10 @@ DefaultDeconvDescs createDescriptorInternalDefault(const dnnl::memory::desc& in_
         attr,
         true);
 
+    // Create dummy primitive to WA CC issue.
+    OPENVINO_ASSERT(dnnl::primitive(conv_desc));
+    OPENVINO_ASSERT(dnnl::primitive(deconv_desc));
```

Comment:
My assumption is that we only need to create a conv primitive in this case, since backward_data primitive will be created later anyway.

Reply:
The assert occurred when it cannot create the chosen pd. 
In conditional compilation(CC) profiling stage, this function ever chose ref_convolution_fwd_t and ref_convolution_bwd_data_t, neither of them will be created finally. But these primitives need be kept by CC so that it can go the same flow with CC profiling stage.

Reply:
We do not really need to have the exact same flow. So, if ref implementation was not used in profiling stage, it is totally fine to not have it available in result. This is basically the main idea of CC feature (to discard unused implementations).
So, i.e., if we used jit_avx512 conv forward primitive to create jit_avx512 conv backward primitive, these are the only primitive descriptors we need to have in the "stripped" binary.
And the dummy conv forward primitive needs to be created as a pair to corresponding conv backward primitive, which was selected for the inference. We don't need to create primitive for every primitive descriptor.

Reply:
ok, update to let CC not keep unnecessary primitive.